src/Microsoft.Diagnostics.Runtime/src/DataTargets/ClrInfo.cs

```diff
@@ -46,7 +46,7 @@ internal ClrInfo(DataTarget dt, ClrFlavor flavor, ModuleInfo module, DacInfo dac
         /// <summary>
         /// If the application is single-file, this contains the runtime, DAC and DBI index information
         /// </summary>
-        public RuntimeInfo? SingleFileRuntimeInfo { get; internal set; }
+        public ClrRuntimeInfo? SingleFileRuntimeInfo { get; internal set; }
```

Comment:
Isn't this a breaking change? SOS/dotnet-dump uses it.

Reply:
Never mind. It won't be a breaking change.  I thought the property name changed for some reason but it is only the type.

Reply:
It **is** a breaking change though, you will be compile-time correct, but not run-time correct.  This is why this version will be shipped as ClrMD `2.1` and not ClrMD `2.0` (the daily builds haven't reflected that version update yet, I mean nuget packages the version will be revved with the other PR I have open for symbols).

I can change this back if it's too much of a hassle.  However, I feel `RuntimeInfo` is way too generic.  I kept hitting it while trying to use [RuntimeInformation](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation?view=net-6.0).  I should have caught that when it was introduced, but I'm fixing it now that we are moving from 2.0 -> 2.1.

Reply:
I don't mind the change at all and it shouldn't break SOS when it gets pushed to the diagnostics repo.  We get clrmd through an DARC update and not from nuget.